test/js/ez-toolbar.js

```diff
@@ -1,11 +1,45 @@
 /* global describe, it, assert, fixture */
 describe('ez-toolbar', function() {
-    it('should be defined', function () {
-        const element = fixture('BasicTestFixture');
+    let element;
 
+    beforeEach(function () {
+        element = fixture('BasicTestFixture');
+    });
+    it('should be defined', function () {
         assert.equal(
             window.customElements.get('ez-toolbar'),
             element.constructor
         );
     });
+
+    describe('properties', function () {
+        describe('`visible`', function () {
+            it('should default to false', function () {
+                assert.notOk(element.visible);
+            });
+
+            describe('set', function () {
+                it('should be reflected to an attribute', function () {
+                    const newVisibility = 'Visible';
+
+                    element.visible = newVisibility;
```

Comment:
setting a string in a Boolean property, something is not correct :) (I'm actually surprised this test is passing btw)